### PR TITLE
minor README update - changed 'n_s' to 'sample_size'

### PR DESCRIPTION
**Pull request by hwilliams10**
changed 'n_s' to 'sample_size'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,11 +153,12 @@ plot_sampling_hist(pop, samples, parameter)
 
 ```python
 import numpy as np
-from samplingsimulatorpy import generate_virtual_pop,
-                                draw_samples,
-                                plot_sample_hist,
-                                plot_sampling_hist,
-                                stat_summary
+from samplingsimulatorpy.generate_virtual_pop import generate_virtual_pop
```
Comment (tguo9) [inline README.md:156]: Looking great in this part, but please do update the previous chunks as well

**Review by tguo9 — CHANGES_REQUESTED** [review]
Please fix the imports for all functions. Thanks

**Comment by hwilliams10** [comment]
> Please fix the imports for all functions. Thanks

Done in latest commit @tguo9

**Review by tguo9 — APPROVED** [review]
Looks good!

**Pull request merged**